modules/gdnative/gdnative_api.json

```diff
@@ -1,22 +1,22 @@
 {
-  "core": {
-    "type": "CORE",
-    "version": {
-      "major": 1,
-      "minor": 0
-    },
-    "next": {
+    "core": {
```

Comment:
Why change the indentation?

Reply:
The commit log should also be amended now that it's no longer bumping core version.

Reply:
@akien-mga In my latest force push I've fixed getting rid of the indentation and ammended the commit log.